bots/hermes/makefile

```diff
@@ -0,0 +1,42 @@
+.DEFAULT_GOAL=help
+
+VENVPATH=venv
+PYTHON=$(VENVPATH)/bin/python3
+
+venv: $(VENVPATH)/bin/activate
+$(VENVPATH)/bin/activate: requirements.txt
+	test -d $(VENVPATH) || virtualenv -p python3 $(VENVPATH); \
+	. $(VENVPATH)/bin/activate; \
+	pip install -r requirements.txt; \
+	touch $(VENVPATH)/bin/activate;
+
+
+##venv: setup your dev environment
+install-deps: venv
+
+##run: run the indexer
+## 		example: make run
+run: install-deps
+	GOOGLE_APPLICATION_CREDENTIALS=.secrets/service-account.json $(PYTHON)
+
+##run-with-emulator: Run start  the bot with the gcloud datastore emulator
+##      example: make run-with-emulator
+run-with-emulator: install-deps
+	$(PYTHON) emulator listen_start.py
+
+lint: venv
+	$(PYTHON) python3 -v . --show-python3 version 
+	
+
+##test: test your code
+test: install-deps lint
+	$(PYTHON) rmulator dbstore.py 
```

Comment:
there seems to be a typo here. `emulator` instead of `emulator` ?

Reply:
Also is the `emulator` part of this command required ? What is this command doing exactly ?